src/labels/CustomLabels.labels

```diff
@@ -7328,6 +7328,126 @@ If preferred, you can disable Gift Entry and instead use older Batch Gift Entry
+        <value>Run Reports</value>
+    </labels>
+    <labels>
+        <fullName>gseuChecklistItemPrebuiltNPSPReportSecBtnLabel</fullName>
+        <categories>Get Started Page</categories>
+        <language>en_US</language>
+        <protected>true</protected>
+        <shortDescription>Button label of checklist item Prebuilt NPSP Report in GS for End User</shortDescription>
+        <value>Reports Workbook</value>
+    </labels>
+    <labels>
+        <fullName>gseuChecklistItemPrebuiltNPSPReportTitle</fullName>
+        <categories>Get Started Page</categories>
+        <language>en_US</language>
+        <protected>true</protected>
+        <shortDescription>Title of checklist item Prebuilt NPSP Report in GS for End User</shortDescription>
+        <value>Run a Prebuilt NPSP Report</value>
+    </labels>
+    <labels>
+        <fullName>gseuChecklistItemStandardReportTypeDesc</fullName>
+        <categories>Get Started Page</categories>
+        <language>en_US</language>
+        <protected>true</protected>
+        <shortDescription>Description of checklist item Standard Report Type in GS for End User</shortDescription>
+        <value>The Lightning Report Builder gives you the ability to create your own custom reports using standard report types. The report type you choose determines the objects and fields available in your report.</value>
+    </labels>
+    <labels>
+        <fullName>gseuChecklistItemStandardReportTypeLinkLabel</fullName>
+        <categories>Get Started Page</categories>
+        <language>en_US</language>
+        <protected>true</protected>
+        <shortDescription>Link label of checklist item Standard Report Type in GS for End User</shortDescription>
+        <value>Learn more about Standard Report Types in Trailhead</value>
+    </labels>
+    <labels>
+        <fullName>gseuChecklistItemStandardReportTypePriBtnLabel</fullName>
+        <categories>Get Started Page</categories>
+        <language>en_US</language>
+        <protected>true</protected>
+        <shortDescription>Button label of checklist item Standard Report Type in GS for End User</shortDescription>
+        <value>Create Reports</value>
+    </labels>
+    <labels>
+        <fullName>gseuChecklistItemStandardReportTypeTitle</fullName>
+        <categories>Get Started Page</categories>
+        <language>en_US</language>
+        <protected>true</protected>
+        <shortDescription>Title of checklist item Standard Report Type in GS for End User</shortDescription>
+        <value>Create a Report from a Standard Report Type</value>
+    </labels>
+    <labels>
+        <fullName>gseuChecklistItemViewDashboardDesc</fullName>
+        <categories>Get Started Page</categories>
+        <language>en_US</language>
+        <protected>true</protected>
+        <shortDescription>Description of checklist item View Dashboard in GS for End User</shortDescription>
```

Comment:
I recommend changing this to: Description of View Dashboard checklist item in Get Started for End Users.

Reply:
@fparodi-salesforce I think you might have missed this comment.